packages/viewer/src/lib/types/measurementUnits.ts

```diff
@@ -46,32 +46,32 @@ class MillimeterUnitConverter extends ScaledUnitConverter {
 }
 
 class CentimeterUnitConverter extends ScaledUnitConverter {
-  public constructor() {
-    super(1 / 100);
+  public constructor(power = 1) {
+    super(1 / Math.pow(10, power));
```

Comment:
The conversion factor for centimeters wasn't correct, changed it to 10 here to fix the mm -> cm conversion for area as well as length.